components/LessonTitleCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import '../scss/lessonTitleCard.scss'
```

Comment:
I did not experienced it myself at work but could you take a look at @use instead of import that will be deprecated soon?
[DOC](https://sass-lang.com/documentation/at-rules/import)
>The Sass team discourages the continued use of the @import rule. Sass will gradually phase it out over the next few years, and eventually remove it from the language entirely. Prefer the @use rule instead.